help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -122,6 +132,7 @@ To configure details for the destination, fill in the required and optional fiel
 
 * **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 * **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+* **[!UICONTROL Customization Prefix]**: The part of the name of any custom field you create in [!DNL Dynamics 365] is the customization prefix. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details.
```

Comment:
"The part of the name of any custom field you create"

I don't really understand what this means. Can you please rephrase this to be clearer?

Reply:
Its the same phrasing as given by MSD so as to not create confusion.

Reply:
Rephrase this as such:

"The [!UICONTROL Customization prefix] of the custom field that you created in [!DNL Dynamics 365]. Refer to the [Create and edit fields](https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1#create-and-edit-fields) section of the [!DNL Dynamics 365] documentation for further details."

Reply:
Also changed the [!UICONTROL Customization prefix] to `Customization prefix` for the description.